mat/cholesky.go

```diff
@@ -182,10 +182,9 @@ func (c *Cholesky) Solve(m *Dense, b Matrix) error {
 	return nil
 }
 
-// TODO(kortschak): Export this as SolveTwoChol.
-// solveTwoChol finds the matrix m that solves A * m = B where A and B are represented
+// SolveTwo finds the matrix m that solves A * m = B where A and B are represented
 // by their Cholesky decompositions a and b, placing the result in the receiver.
-func (m *Dense) solveTwoChol(a, b *Cholesky) error {
+func (a *Cholesky) SolveTwo(m *Dense, b *Cholesky) error {
```

Comment:
Should this be `SolveChol` instead? Seems more evocative than `SolveTwo`, which means less now.